### ISSUE TITLE
Load all processes from configuration even when starting specific ones

### ISSUE THREAD

**Issue by sciyoshi** [F1bonacc1/process-compose#123]
## Defect

When starting specific processes, the remaining processes should be able to be started.

- [x] Included the relevant configuration snippet
- [x] Included the relevant process-compose log (log location: `process-compose info`)
- [x] Included a [Minimal, Complete, and Verifiable example] (https://stackoverflow.com/help/mcve)

#### Version of `process-compose`:

v0.77.6

#### OS environment:

Darwin aarch64

#### Steps or code to reproduce the issue:

```
processes:
  p1:
    command: "sleep infinity"
  p2:
    command: "sleep infinity"
```

Run `process-compose up p1`.

#### Expected result:

I should now be able to start p2 from the TUI or remote client.

#### Actual result:

Only p1 is available in the UI.

**Comment by F1bonacc1** [comment]
Hey @sciyoshi,

Thanks for trying process-compose.

What is the desired outcome in this case? The selected processes are running and the rest are disabled?

**Comment by sciyoshi** [comment]
Yes, ideally starting a subset of processes would simply disable the other ones, not remove them from the config entirely.

**Comment by F1bonacc1** [comment]
Fixed in v0.77.8

**Comment by sciyoshi** [comment]
Awesome, thank you for the quick response + release! Much appreciated 👍
